### PR TITLE
Add a set of release notes for MS1

### PR DESCRIPTION
**Pull request by SueChaplain**
Draft release notes for 0.15, milestone 1.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

File: doc/release-notes/0.15/0.15.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/164">#164</a></td>
+<td valign="top">ChaCha20 algorithm support </td>
+<td valign="top">OpenJDK11 only</td>
+<td valign="top">On Java 11, ChaCha20 and ChaCha20-Poly1305 algorithms can now use OpenSSL. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/54">#54</a></td>
+<td valign="top">OpenJ9 now supports all POSIX signals listed in [SunMiscSignalTest.java](https://github.ibm.com/runtimes/openj9-openjdk-jdk/blob/openj9/test/jdk/sun/misc/SunMiscSignalTest.java)</td>
```
Comment (pshipton) [inline doc/release-notes/0.15/0.15.md]: The link is to an ibm internal repo, use https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/test/jdk/sun/misc/SunMiscSignalTest.java

File: doc/release-notes/0.15/0.15.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+<td valign="top">Compatibility support for -XX:OnOutOfMemoryError</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">For compatibility with HotSpot, this option can now be used to run a command or list of commands when a
+`java.lang.OutOfMemoryError` exception occurs. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/5239">#5293</a></td>
+<td valign="top">Support for automatic huge pages allocation</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">The VM now supports automatic allocation of huge pages on Linux when you use the `madvise` setting. </td>
```
Comment (pshipton) [inline doc/release-notes/0.15/0.15.md]: ... for the transparent_hugepage kernel setting.

**Comment by dchopra001** [comment]
@SueChaplain Watched fields support for zLinux and z/OS was merged in May here:
https://github.com/eclipse/openj9/pull/4743

Can we include this in the release notes?

File: doc/release-notes/0.15/0.15.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+- [OpenJDK 12 with OpenJ9](https://adoptopenjdk.net/archive.html?variant=openjdk12&jvmVariant=openj9)
+
+All builds are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests at AdoptOpenJDK.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.14. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.14/).
```
Comment (PascalSchumacher) [inline doc/release-notes/0.15/0.15.md]: I guess both instances of `0.14` should be `0.15`?

Comment (SueChaplain) [reply doc/release-notes/0.15/0.15.md]: Thanks @PascalSchumacher - both instances fixed.

File: doc/release-notes/0.15/0.15.md (hunk 2 above)
Comment (SueChaplain) [reply doc/release-notes/0.15/0.15.md]: Updated in line with the changes for https://github.com/eclipse/openj9-docs/pull/302. Let me know if you any further tweaks.

File: doc/release-notes/0.15/0.15.md (hunk 1 above)
Comment (SueChaplain) [reply doc/release-notes/0.15/0.15.md]: Fixed

**Comment by SueChaplain** [comment]
>Watched fields support for zLinux and z/OS was merged in May here:
>#4743

>Can we include this in the release notes?

@dchopra001 - Can we get an issue opened at the openj9-docs repo to cover this please? Ideally there would have been a label on the issue or PR to flag that this work was needed. I also spotted https://github.com/eclipse/openj9/pull/6234 for POWER support. Maybe the issue could cover that too ... or a separate issue if it will target a different OpenJ9 release.

Here is a template: https://github.com/eclipse/openj9-docs/issues/new?template=new-documentation-change.md

**Comment by dchopra001** [comment]
> @dchopra001 - Can we get an issue opened at the openj9-docs repo to cover this please? Ideally there would have been a label on the issue or PR to flag that this work was needed

Apologies for the delay on this! I have opened an issue here for Z:
https://github.com/eclipse/openj9-docs/issues/304

I'll let the power team create their own issue once they are ready.

**Comment by gita-omr** [comment]
We would need exactly the same note for Power when https://github.com/eclipse/openj9/pull/6234 is merged, except that I would say "improved" => "improved performance" of field watch support...

**Comment by SueChaplain** [comment]
>We would need exactly the same note for Power when #6234 is merged, except that I would say "improved" => "improved performance" of field watch support...

@gita-omr - please open a docs issue. Here is a template: https://github.com/eclipse/openj9-docs/issues/new?template=new-documentation-change.md

File: doc/release-notes/0.15/0.15.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+<td valign="top"><tt>-Xdiagnosticscollector</tt> option removed</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">No impact. Redundant option removed.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/5374">#5374</a></td>
+<td valign="top">New heuristics to implement compaction during idle garbage collection.</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">Heuristics are added to automatically compact the heap during idle garbage collection (GC) processing. Therefore, if
+<tt>-XXIdleTuningGCOnIdle</tt> is enabled, compaction might occur. To force compaction every time idle GC operations take place,
```
Comment (pshipton) [inline doc/release-notes/0.15/0.15.md]: The options are `-XX:+IdleTuningGCOnIdle` and `-XX:+IdleTuningCompactOnIdle`

Comment (SueChaplain) [reply doc/release-notes/0.15/0.15.md]: Fixed

**Pull request merged**